tools/fado/meson.build

```diff
@@ -0,0 +1,34 @@
+# TODO add these flags
+#ifneq ($(ASAN),0)
+#  CFLAGS    += -fsanitize=address -fsanitize=pointer-compare -fsanitize=pointer-subtract -fsanitize=undefined
```

Comment:
This can be a core option `-Db_sanitize=address,undefined`, no need to specially handle it.